### DISCUSSION TITLE
Shuttle vs relay

### DISCUSSION THREAD

**Discussion by TobleroneSwordfish** [stonepillars/goon-flock#239] [Ideas]
Proposal: the shuttle shouldn't be able to arrive while the relay is active. Give some messages about how the shuttle can't approach due to signal interference and either recall it or keep extending the timer.

**Comment by stonepillars** [comment]
Since the relay explosion absolutely destroys the shit out of the shuttle, probably the following:

- If there is a constructed relay, the shuttle cannot be called due to signal interference
- If the shuttle is _already called_ but has **not arrived** to the station yet, it will be recalled
- If the relay is constructed while the shuttle is at the station, nothing should happen since the shuttle would leave before the relay activates

**Reply by stonepillars** [reply]
Alternative: The shuttle can be called, but it basically forfeit and gives the win to the flock if they have the relay up, and a draw(?) if they don't. Wack.

**Comment by TobleroneSwordfish** [comment]
#246
